### PR TITLE
Move Run A Testsuite on Device Tutorial to VEE Port Guide

### PR DESCRIPTION
**Pull request by acolleux-microej**
# CHANGES

image

### CODE REVIEW

File: VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst [hunk 1]
```diff
@@ -1,69 +1,52 @@
-.. _tutorial_run_test_suite_on_device:
+.. _run_fs_test_suite_on_esp32_wrover:
 
-Run a Test Suite on a Device
-============================
+Run FS Test Suite on ESP32-WROVER VEE Port
```
Comment (gharelMicroEJ) [inline VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst:3]:
```suggestion
Run the FS Test Suite on ESP32-WROVER VEE Port
```

File: VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst [hunk 2]
```diff
@@ -1,69 +1,52 @@
...
 <https://github.com/MicroEJ/VEEPortQualificationTools>`_.
 
-In this tutorial, the target device is the Espressif ESP32-WROVER-KIT
+The target device is the Espressif ESP32-WROVER-KIT
 V4.1 board and the Filesystem Test Suite for :ref:`FS <pack_fs>`
 module will be used as an example.
 
-The tutorial should take 1 hour to complete (excluding the Platform
+This guide should take 1 hour to complete (excluding the VEE Port
 Getting Started setup).
```
Comment (gharelMicroEJ) [inline VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst:16]:
```suggestion
setup fron the Getting Started page).
```

File: VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst [hunk 3]
```diff
@@ -1,69 +1,52 @@
...
+- MICROEJ SDK distribution 20.07 or more (see :ref:`get_sdk_version`).
 
-- The `WROVER Platform
+- The `WROVER VEE Port
   <https://github.com/MicroEJ/Platform-Espressif-ESP-WROVER-KIT-V4.1/tree/1.6.2>`__
   has been properly setup (i.e., it can be used to generate a Mono-Sandbox Executable).
 
-The explanation can be adapted to run the test suite on any other
-MicroEJ Platform providing:
+The explanation can be adapted to run the test suite on any other VEE Port providing:
```
Comment (gharelMicroEJ) [inline VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst:33]:
```suggestion
The instructions of this page can be adapted to run the test suite on any other VEE Port providing:
```

**Review by gharelMicroEJ — APPROVED** [review]

File: VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst (hunk 2 above)
Comment (acolleux-microej) [reply VEEPortingGuide/runFsTestSuiteOnESP32WROVER.rst]: Fixed

**Pull request merged**